cardinal/engine_test.go

```diff
@@ -421,22 +422,19 @@ func TestTransactionsSentToRouterAfterTick(t *testing.T) {
 	tf.DoTick()
 }
 
-// setEnvToCardinalProdMode sets a bunch of environment variables that are required
-// for Cardinal to be able to run in Production Mode.
-func setEnvToCardinalProdMode(t *testing.T) {
-	t.Setenv("CARDINAL_MODE", string(cardinal.RunModeProd))
-	t.Setenv("REDIS_ADDRESS", "localhost:6379")
-	t.Setenv("REDIS_PASSWORD", "bar")
-	t.Setenv("CARDINAL_NAMESPACE", "baz")
-	t.Setenv("BASE_SHARD_SEQUENCER_ADDRESS", "localhost:8080")
-	t.Setenv("ROUTER_KEY", "77cf59146831dbd94bd19dd4b259b268ee07a7c1fdba67e92b0f7c1cfdfb7a9b")
+// setEnvToCardinalRollupMode sets a bunch of environment variables that are required
+// for Cardinal to be able to run in rollup node.
+func setEnvToCardinalRollupMode(t *testing.T) {
+	t.Setenv("CARDINAL_ROLLUP_ENABLED", strconv.FormatBool(true))
```

Comment:
`strconv.FormatBool(true)` could just be `"true"`

Reply:
![CleanShot 2024-04-15 at 15 35 49@2x](https://github.com/Argus-Labs/world-engine/assets/4402697/28525dd4-1e16-4896-b361-814459ad81f8)
dont think so

Reply:
oh i didnt see that you want me to shove in a true string. i'd prefer not to do that, seems typo prone and since it's a test, correctness matters more than slight performance cost

Reply:
idk seems fine since we've hardcoded all the first arguments to t.Setenv here

Reply:
in absence of strong reason, i'd prefer to just keep it as is